LiteCore/Storage/KeyStore.hh

```diff
@@ -75,6 +75,8 @@ namespace litecore {
         /// Set of characters allowed in a collection or scope name:
         static constexpr slice kCollectionNameCharacterSet
                             = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ01234567890_-%";
+        // Separator for an FTI keystore (i.e. <scope>.<collection>::<index>
+        static constexpr char kIndexSeparator = ':';
```

Comment:
Is it "::" ?

Reply:
Yeah, but because I did search for first colon, then second, I had to use ‘:’ twice. Per Jim’s suggestion though, I will adjust it to simply search for “::” instead of ‘:’ then ‘:’ again